Explorer/Assets/DCL/Backpack/Gifting/Models/EmoteViewModel.cs

```diff
@@ -0,0 +1,63 @@
+﻿using DCL.Backpack.Gifting.Models;
+using UnityEngine;
+
+public readonly struct EmoteViewModel : IGiftableItemViewModel
```

Comment:
There is no difference between `WearableViewModel` and `EmoteViewModel` - they expose right the same abstract type, and thus can be unified into a single structure, and the interface can be removed